runtime/src/bank.rs

```diff
@@ -4329,7 +4327,12 @@ impl Bank {
 
         debug!("simulate_transaction: {:?}", timings);
 
-        let execution_result = execution_results.pop().unwrap();
+        let execution_result =
+            execution_results
+                .pop()
+                .unwrap_or(TransactionExecutionResult::NotExecuted(
+                    TransactionError::InvalidProgramForExecution,
```

Comment:
i guess this is about as generic a `TransactionError` as we have :thinking: 

prolly fine for now since this is isolated to simulation

Reply:
Yeah, the other one we could use was `AccountInUse`. Nothing generic otherwise.

Reply:
assuming by "replay" you mean that the tx is retryable, no. when a tx makes it to execute, it's getting fees taken and committed in a block. even if it were, this logic is in simulate, which won't be committed anyway

Reply:
I meant retryable yes. And I forgot that this is simulation and thus a failure is not repeated. So seems alright to me.